### PR TITLE
Made the default build untested

### PR DESCRIPTION
**Pull request by TiddoLangerak**
Previously `npm build` would also run all tests. However, `npm build` is installed on pre-publish, meaning that it will also always be run on `npm install`. Usually we don't want all tests here (e.g. Jenkins breaks), so `npm build` now doesn't test anymore.

@Magnetme/developers - Ready for review

### CODE REVIEW

**Comment by TiddoLangerak** [comment]
And with this branch we also have Jenkins tests!

**Comment by wouterwillems10** [comment]
I dont fully understand your explanation why the tests should be removed when `npm build`. Can you rephrase?

**Comment by TiddoLangerak** [comment]
I made a mistake, should be `gulp build` not `npm build`. But the reason is that often when you want to build the project you don't actually want to run all tests. E.g. when the package is installed `gulp build` will run to create the bundled packages, but you don't want to test everything all the time, especially since this is also run when you install it as a dependency. Additionally the test task can only succeed on computers that have Chrome, Firefox and PhantomJS since it runs tests in all of them. This resulted in Jenkins not being able to install the package at all. 

So now the build step doesn't automatically run tests. You can of course still run the tests with `gulp test`.

**Comment by rogierslag** [comment]
:+1:

**Comment by wouterwillems10** [comment]
ah that makes sense
